CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ All notable changes to this project will be documented in this file.
 #### Anomalies
 
 * Fixes ambiguity issue with  `Single.do(onNext:onError:onSubscribe:onSubscribed:onDispose:)` and `Single.do(onSuccess:onError:onSubscribe:onSubscribed:onDispose:)`.
+* Fixes misspelled `immediatelly` to `immediately`, `Programatic` to `Programmatic`.
+* Adds missing `scheduler` parameter.
```

Comment:
Can you make these two into one? 

* Fixes various spelling issues and missing parameters.

Reply:
think you, I resubmitted.